.gitlab-ci.yml

```diff
@@ -87,3 +115,13 @@ release_rpm_x64:
   extends: .release
   variables:
     IMAGE: rpm_x64
+
+release_windows_x64:
+  extends: .release
+  variables:
+    IMAGE: windows_x64
+
+release_windows_x86:
+  extends: .release
+  variables:
+    IMAGE: windows_x86
```

Comment:
Someone else can confirm, but I don't think that will work: we're trying to pull a Windows image on a Linux host, which will probably lead to the following error:
```
image operating system "windows" cannot be used on this platform
```

We probably need to reintroduce a Windows-specific step to push the Windows containers.